### PR TITLE
fix(outdated): show a suggestion for updating

### PR DESCRIPTION
**Pull request by bartlomieju**
Show command to update dependencies based on the output
of "deno outdated" command.

Screenshot 2024-12-10 at 03 10 31

Closes https://github.com/denoland/deno/issues/27256

### CODE REVIEW

**Review by nathanwhit — APPROVED** [review]
LGTM

File: cli/tools/registry/pm/outdated.rs [hunk 1]
```diff
@@ -100,6 +100,24 @@ fn print_outdated_table(packages: &[OutdatedPackage]) {
   println!("└{package_fill}┴{current_fill}┴{update_fill}┴{latest_fill}┘",);
 }
 
+#[allow(clippy::print_stdout)]
```
Comment (dsherret) [inline cli/tools/registry/pm/outdated.rs:103]: Maybe we should use `log::info` for this?

**Review by dsherret — APPROVED** [review]
LGTM, but I think we should use `log::info` instead.

**Comment by bartlomieju** [comment]
> LGTM, but I think we should use `log::info` instead.

I believe this is the same situation as with `deno lint --rules` which has this comment:
> // The rules should still be printed even if `--quiet` option is enabled,
// so use `println!` here instead of `info!`.

So I think it's fine to use `println!` here

**Comment by dsherret** [comment]
I don't think this is vital to the main information being displayed though.

**Pull request merged**